common/utils/get-license-info.js

```diff
@@ -65,36 +59,34 @@ export const licenseMap = {
       'Creative Commons Attribution Non-Commercial No-Derivatives (CC BY-NC-ND 4.0) terms and conditions <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">https://creativecommons.org/licenses/by-nc-nd/4.0/</a>',
     ],
   },
-  inc: {
-    url: 'http://rightsstatements.org/vocab/InC/1.0/',
-    text: 'In copyright',
-    description:
-      'Free to use as permitted by the copyright and rights legislation that applies to your use.',
-    humanReadableText: [
-      'It is possible this Item is protected by copyright and/or related rights. You are free to use this Item in any way that is permitted by the copyright and related rights legislation that applies to your use. For other uses you need to obtain permission from the rights-holder(s).',
-    ],
+  'cc-by-nd': {
+    url: 'https://creativecommons.org/licenses/by-nd/4.0/',
+    text: 'CC BY-ND',
+    icons: ['cc', 'ccBy', 'ccNd'],
+    description: 'Free to use with attribution. No modifications permitted.',
+  },
+  'cc-by-sa': {
+    url: 'https://creativecommons.org/licenses/by-sa/4.0/',
+    text: 'CC BY-SA',
+    icons: ['cc', 'ccBy'],
```

Comment:
Out of curiosity, why the change to camel case in the name of the icons @gestchild? As far as I can see the images name is with underscores?

Reply:
These are the names that get passed to the `<Icon />` component and should correspond to the ones here: https://github.com/wellcometrust/wellcomecollection.org/blob/bce3e7970038e2b67f7e97a3efea6ae53a78f798/common/icons/index.js#L17